app/src/main/java/org/alexcode/snake/Settings.java

```diff
@@ -0,0 +1,119 @@
+package org.alexcode.snake;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.CompoundButton;
+import android.widget.Spinner;
+import android.widget.Switch;
```

Comment:
maybe import android.widget.*?

Reply:
Ok, but I don't know if is a good idea to import all subfolders for some from the point of view of resource usage.